src/test/java/org/traccar/protocol/HuabaoProtocolDecoderTest.java

```diff
@@ -11,6 +11,14 @@ public void testDecode() throws Exception {
 
         var decoder = inject(new HuabaoProtocolDecoder(null));
 
+        verifyAttribute(decoder, binary(
```

Comment:
One test is probably enough. Let's just keep negative temperature. It's a better test IMO.